src/main/java/com/mparticle/kits/RevealMobileKit.java

```diff
@@ -25,35 +27,46 @@
  *  - ./src/main/AndroidManifest.xml
  *  - ./consumer-proguard.pro
  */
-public class ExampleKit extends KitIntegration {
+public class RevealMobileKit extends KitIntegration {
+
+    public Reveal revealSDK = null;
 
     @Override
     protected List<ReportingMessage> onKitCreate(Map<String, String> settings, Context context) {
-        /** TODO: Initialize your SDK here
-         * This method is analogous to Application#onCreate, and will be called once per app execution.
-         *
-         * If for some reason you can't start your SDK (such as settings are not present), you *must* throw an Exception
-         *
-         * If you forward any events on startup that are analagous to any mParticle messages types, return them here
-         * as ReportingMessage objects. Otherwise, return null.
-         */
+        //Create Reveal configuration.
+        this.revealSDK = Reveal.getInstance();
+        String apiKey = settings.get( "apiKey" );
+        String serviceTypeString = settings.get( "serviceType" );
+
+        this.revealSDK.setDebug( true );
+
+        if ( apiKey != null ) {
+            Reveal.ServiceType serviceType = Reveal.ServiceType.PRODUCTION;
+
+            if (serviceTypeString != null) {
+                serviceTypeString = serviceTypeString.toLowerCase();
+
+                if (serviceTypeString.equals("sandbox"))
```

Comment:
the values of these settings are in our control, so we can make sure it's always 'sandbox' for both iOS and Android.

That said, would it make sense to just always use sandbox when the mParticle SDK is in dev mode, similar to setting debug mode above?